src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+
+    @Override
+    public void doWork() {
+        System.out.println("Truck started its work.");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Truck stopped working.");

```

Comment:
Similar to the `doWork` method, the message in the `stopWork` method should be more informative. Consider rephrasing it to something like 'Truck has stopped working.' to clearly indicate that it's a Truck.